code/modules/surgery/organs/augments_internal.dm

```diff
@@ -291,7 +291,7 @@
 		owner.unEquip(holder, 1)//if he does, take it away.
 		holder.loc = null
 		out = 0
-		owner.visible_message("<span class='notice'>[holder] forcibly retracts into your arm.</span>")
+		owner << "<span class='warning'>[holder] forcibly retracts into your arm.</span>"
```

Comment:
Sort of. It's not a big deal though. It's similar to getting "you feel woozy" while dead and low on health, not very realistic.